sphinx/themes/basic/theme.conf

```diff
@@ -7,7 +7,7 @@ sidebars = localtoc.html, relations.html, sourcelink.html, searchbox.html
 [options]
 nosidebar = false
 sidebarwidth = 230
-body_min_width = 450
+body_min_width = 300
```

Comment:
Is any reason to choose this value?

I just searched the screensizes of well-used mobile devices. Some reports says 360px is smallest size at present.
https://www.browserstack.com/guide/ideal-screen-sizes-for-responsive-design
https://mediag.com/blog/popular-screen-resolutions-designing-for-all/

Reply:
I picked it because it solved the bug. Another value could work fine it also resolve the issue.